stl/inc/ranges

```diff
@@ -1524,15 +1524,19 @@ namespace ranges {
 
     namespace views {
         template <class _Rng>
+#ifdef __EDG__ // TRANSITION, DevCom-10698021
+        concept _Can_as_rvalue = requires(_Rng&& __r) { as_rvalue_view(static_cast<_Rng&&>(__r)); };
+#else // ^^^ workaround / no workaround vvv

```

Comment:
Not sure whether "workaround" is appropriate here as the standard wording does say direct-initialization ([[range.as.rvalue.overview]/2.2](https://eel.is/c++draft/range.as.rvalue.overview#2.2)).

Reply:
Seems like we should perma-workaround this by Doing What The Standard Says<sup>TM</sup>.